### ISSUE TITLE
Can't add more than one value to many-to-many fields

### ISSUE THREAD

**Issue by fatihky** [directus/next#697]
Hi,
Currently, I can't add multiple values to a many-to-many (m2m) fields. I can only one value and these happen:
1. Other values get ignored if I select multiple values when the field is empty
2. Newly selected value doesn't get added to the list if I select another value when the field has a value.

Steps to reproduce:
1. Create two models and add a many-to-many relationship field to one of the models. For example, `articles` and `tags`
2. Create two tags.
3. Create a new article and try to select both of the tags that you've created.

Update:

I've just discovered a few more things. For example, `/items/articles/1` endpoint returns this response:
```json
{
  "data": {
    "id": 1,
    "date_created": "2020-10-17T21:51:26+03:00",
    "title": "article 1",
    "single_tag": 1,
    "tags": [
      1,
      2,
      3,
      4,
      5
    ]
  }
}
```
I think the `tags` field has multiple values because I've tried to add tags to the article multiple times. It seems they were added to the article, but they just weren't shown on UI.

So the problems are:
1. I can add the same tag to the article multiple times. As a result, multiple relation rows generated for the same article-tag pair.
2. I can't see all the selected tags on UI. I can only see one. So I thought tags didn't get added. But it seems they were added.
3. After deleting tags from the article, `articles_tags` rows don't get removed. Instead, their `articles_id` fields are set to `NULL`.

**Comment by Nitwel** [comment]
> After deleting tags from the article, articles_tags rows don't get removed. Instead, their articles_id fields are set to NULL.

Just so you know, this is intentional. We a looking forward into adding an option to delete the junction item instead of settings it's value to null but for now it does that because on o2m you don't want to delete the related item normally.

**Comment by Nitwel** [comment]
This might have been resolved by 16cd8838a9fd2f94617f1818ba4060fb113146af
If this bug still occurs to you in the next release, please reopen this issue again so we can look further into it.

**Comment by fatihky** [comment]
Thank you @Nitwel, I'll let you know if this occurs again.
